specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/blob.json

```diff
@@ -0,0 +1,1542 @@
+          },
+          {
+            "$ref": "#/parameters/ContainerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/LeaseContainerRequest"
+            },
+            "description": "Lease Container request body."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Lease Container operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/LeaseContainerResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/migrate": {
+      "put": {
+        "tags": [
+          "BlobContainers"
+        ],
+        "operationId": "BlobContainers_ObjectLevelWorm",
+        "x-ms-examples": {
+          "VersionLevelWormContainerMigration": {
+            "$ref": "./examples/ObjectLevelWormContainerMigration.json"
+          }
+        },
+        "description": "This operation migrates a blob container from container level WORM to object level immutability enabled container. Prerequisites require a container level immutability policy either in locked or unlocked state, Account level versioning must be enabled and there should be no Legal hold on the container.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "#/parameters/ContainerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- returned when the blob container migration to support object level immutability completes."
+          },
+          "202": {
+            "description": "Accepted -- Blob Container migration to support object level immutability is accepted; operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Do we also need add "x-ms-long-running-operation-options" tag. 
Per [this doc](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation-options): 

> When x-ms-long-running-operation is specified, there should also be a x-ms-long-running-operation-options specified.

Reply:
It's fine to take defaults.